### DISCUSSION TITLE
X drip and G7

### DISCUSSION THREAD

**Discussion by SEBAM107** [NightscoutFoundation/xDrip#3201] [Q&A]
Hi all, I am trying to instal xdrip on my Android and connect it to my sons G7. Could anybody help please? I tried to setup but G7 doesnt have a transmitter ID - or I could not find it in the package. Then I used hardware data source: companion app. It seemd to be warming up. Then it didn't work 😞 what am I doing wrong? Thank you

**Comment by Navid200** [comment]
I have explained it here: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

Please read that.  Then, if anything is not clear, please let me know.

**Comment by SEBAM107** [comment]
It worked! Thank you. Lifesaver

**Reply by Navid200** [reply]
Awesome!
